### PR TITLE
ODC-5596-Updated cypress extensibility related files

### PR DESCRIPTION
**Pull request by makambalaji**
- Update the pacakge.json file with below scripts

> test-cypress-pipelines-headless & test-cypress-pipelines
> test-cypress-knative-headless & test-cypress-knative
> test-cypress-gitops-headless & test-cypress-gitops

- Update the .gherkinlintrc file 

> updated knative related tags
> max scenarios per file
> file name style

- Updated the OWNERS file for gitops, knative and pipelines

### CODE REVIEW

**Comment by makambalaji** [comment]
VeereshAardhaya raised a request to join openshift.org, I think approval is not done yet. due to that reason `do-not-merge/invalid-owners-file` label is added.until then marking this pr as hold
/hold

**Review by andrewballantyne — APPROVED** [review]

File: frontend/package.json [hunk 1]
```diff
@@ -39,6 +39,12 @@
...
+    "test-cypress-pipelines-headless": "cd packages/pipelines-plugin/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/*/*.feature\";",
+    "test-cypress-gitops": "cd packages/gitops-plugin/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",
+    "test-cypress-gitops-headless": "cd packages/gitops-plugin/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/*/*.feature\";",
+    "test-cypress-knative": "cd packages/knative-plugin/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",
+    "test-cypress-knative-headless": "cd packages/knative-plugin/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/*/*.feature\";",
```
Comment (dtaylor113) [inline frontend/package.json:47]: Hi, are any of these supposed to run during CI?

Comment (makambalaji) [reply frontend/package.json:47]: Yes Dave, below are the scripts needs to be executed on CI, but I don't want to run them on all pr's
test-cypress-gitops-headless [This should run, only when gitOps label is added to the pr]
test-cypress-knative-headless  [This should run, only when knative label is added to the pr]
I will raise this ci implementation as separate pr

**Comment by makambalaji** [comment]
/hold cancel
Removing hold, as 'do-not-merge/invalid-owners-file' label is already added

**Comment by VeereshAradhya** [comment]
Created a ticket https://issues.redhat.com/browse/DPP-6910

**Pull request merged**